tests/by-util/test_date.rs

```diff
@@ -43,6 +43,14 @@ fn test_date_rfc_3339() {
     }
 }
 
+
```

Comment:
please run runfmt

Reply:
[This](https://github.com/uutils/coreutils/pull/4535/commits/1eb9299c780fc35d9d1fdcaa0c233abe75f0899a) commit should fix it.